src/main/java/com/dnd/reetplace/app/dto/place/response/PlaceSearchResponse.java

```diff
@@ -0,0 +1,186 @@
+            "<li>ACTIVITY_PARK - 공원</li>" +
+            "<li>ACTIVITY_ROLLER - 롤러장</li>" +
+            "<li>ACTIVITY_COIN_KARAOKE - 코인노래방</li>" +
+            "<li>PHOTO_LIFE_FOUR_CUT - 인생네컷</li>" +
+            "<li>PHOTO_PHOTO_SIGNATURE - 포토시그니처</li>" +
+            "<li>PHOTO_HARU_FILM - 하루필름</li>" +
+            "<li>PHOTO_SIHYUN_HADA - 시현하다 프레임</li>" +
+            "<li>PHOTO_MONO_MANSION - 모노맨션</li>" +
+            "<li>PHOTO_RGB_PHOTO - RGB 포토</li>" +
+            "<li>PHOTO_PHOTOISM - 포토이즘</li>" +
+            "<li>SHOPPING_DEPARTMENT_STORE - 백화점</li>" +
+            "<li>SHOPPING_MART - 마트</li>" +
+            "<li>SHOPPING_MARKET - 시장</li>" +
+            "<li>CAFE - 카페</li>" +
+            "<li>CAFE_BOOK - 북카페</li>" +
+            "<li>CAFE_CARTOON - 만화카페</li>" +
+            "<li>CAFE_DESERT - 디저트카페</li>" +
+            "<li>CAFE_FRESH_FRUIT - 생과일전문점</li>" +
+            "<li>CULTURE_CINEMA - 영화관</li>" +
+            "<li>CULTURE_CAR_CINEMA - 자동차극장</li>" +
+            "<li>CULTURE_CONCERT - 공연장,연극극장</li>" +
+            "</ul>",
+            example = "ACTIVITY_BOARD_GAME")
+    private PlaceSubCategory subCategory;
+
+    @Schema(description = "대표 전화 번호", example = "02-363-3799")
+    private String phone;
+
+    @Schema(description = "지번 주소", example = "서울 서대문구 창천동 18-31")
+    private String lotNumberAddress;
+
+    @Schema(description = "도로명 주소", example = "서울 서대문구 연세로 8")
+    private String roadAddress;
+
+    @Schema(description = "위도", example = "37.5561776340198")
+    private String lat;
+
+    @Schema(description = "경도", example = "126.93713158887188")
+    private String lng;
+
+    @Schema(description = "<p>북마크 종류. 목록은 다음과 같음</p>" +
+            "<ul>" +
+            "<li>WANT - 가보고 싶어요</li>" +
+            "<li>GONE - 다녀왔어요</li>" +
+            "<li>북마크가 없을 시 null</li>" +
+            "</ul>",
+            example = "WANT")
+    private BookmarkType type;
+
+    @Schema(description = "북마크 고유 id값 (북마크 없을 시 null)", example = "1")
+    private Long bookmarkId;
+
+    @Schema(description = "릿플점수 (북마크 없을 시 null)", example = "3")
+    private Short rate;
+
+    public static PlaceSearchResponse of(
+            KakaoPlaceSearchResponse kakaoResponse,
+            BookmarkType bookmarkType,
+            Long bookmarkId,
+            Short rate
+    ) {
+        PlaceCategoryGroupCode categoryGroupCode;
+        try {
+            categoryGroupCode =
+                    PlaceCategoryGroupCode.valueOf(kakaoResponse.getCategory_group_code());
+        } catch (Exception e) {
+            categoryGroupCode = null;
+        }
+        PlaceCategory category = null;
+        PlaceSubCategory subCategory = null;
+
+        for (PlaceSubCategory sub : PlaceSubCategory.values()) {
+            if (kakaoResponse.getCategory_name().contains(sub.getDescription())) {
+                subCategory = sub;
+                category = sub.getMainCategory();
+                break;
+            }
+        }
```

Comment:
이 부분 반복문으로 일일히 찾는 것보단 filter를 쓰면 어떨까요?
```suggestion
        PlaceSubCategory subCategory = Arrays.stream(PlaceSubCategory.values())
                .filter(sub -> kakaoResponse.getCategory_name().contains(sub.getDescription()))
                .findFirst()
                .orElse(null);
        PlaceCategory category = subCategory == null ? null : subCategory.getMainCategory();
```